### PR TITLE
Tests and fixes for permissions routes

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
@rudigiesler depends on #63, otherwise ready for review

**Comment by justinvdm** [comment]
Found #64 and #65 while working on this, will fix those outside of this PR.

**Comment by justinvdm** [comment]
Ready for review.

File: src/contexts/permission.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ function createAccess(teamId, permission) {
 
 function removeAccess(teamId, id, {auth}) {
   return authApi.teams.get(teamId, {conf: authConf(auth)})
-    .then(({data: {permissions}}) => find(permissions, {id}))
+    .then(({data: {permissions}}) => find(permissions, {id: +id}))
```
Comment (rudigiesler) [inline src/contexts/permission.js:20]: I'm not sure why this is `+id`?

Comment (rudigiesler) [inline src/contexts/permission.js:20]: Ah, it's a way in node to change a string to an int. Nevermind.

**Comment by rudigiesler** [comment]
:+1:

File: src/contexts/permission.js (hunk 1 above)
Comment (justinvdm) [inline src/contexts/permission.js:20]: Yep, planning on removing the `+` once we are getting string ids.

**Pull request merged**